tests/unit/testtasks.cpp

```diff
@@ -104,8 +104,58 @@ void TestTasks::function() {
   QVERIFY(completed);
 }
 
-void TestTasks::removeDevice() {
```

Comment:
Would you mind filing a ticket as a reminder that we need to write removeDevice task tests?

Reply:
https://mozilla-hub.atlassian.net/browse/VPN-2896